### PR TITLE
Adding the ability to use serial without primary key

### LINKED ISSUE #114: 🐛 It seems that there is an issue with changing primary keys when you try to add sequenced type.
### No duplicates 🥲.

- [X] I have searched for a similar issue in our bug tracker and didn't find any solutions.

### What happened?

I have the table:

```
class MyTable
{
    #[Column(type: 'uuid', name: 'uuid', primary: true)]
    private string $id;
}

```
When trying to add a new column to this table using the following migration command:

```
public function up(): void
{
    $this->database()->execute("ALTER TABLE my_table ADD COLUMN my_column SERIAL;")
}

```

I receive the following error when running "php ./app.php cycle:sync":

```
Detecting schema changes:
• my_db.my_table: 1 change(s) detected
[Cycle\Schema\Exception\SyncException]
Unable to change primary keys for existed table
in vendor/cycle/schema-builder/src/Generator/SyncTables.php:47

Previous: [Cycle\Database\Exception\DBALException]
Unable to change primary keys for existed table
in vendor/cycle/database/src/Driver/Handler.php:66

```
Also expression like
`#[Column(type: 'bigPrimary', name: 'm_index', primary: false)]
`
not working too.

Thank you.

### Version

```shell
ORM v2.2.2
PHP 8.1
```

### PR DESCRIPTION
**Pull request by msmakouz**
Related to the: https://github.com/cycle/database/issues/114

> **Note**:
> Documentation needed

### CODE REVIEW

File: src/Driver/Postgres/Schema/PostgresColumn.php [hunk 1]
```diff
@@ -514,12 +515,11 @@ public static function createInstance(
                 $column->type === 'smallint' => 'smallserial',
                 default => 'serial'
             };
-            $column->autoIncrement = true;
```
Comment (roxblnfk) [inline src/Driver/Postgres/Schema/PostgresColumn.php]: It need to back. We will delete it with the `autoIncrement` property later

File: src/Driver/Postgres/Schema/PostgresColumn.php [hunk 2]
```diff
@@ -248,6 +274,8 @@ class PostgresColumn extends AbstractColumn
     #[ColumnAttribute(['interval'])]
     protected ?string $intervalType = null;
 
+    protected bool $isPrimary = false;
```
Comment (roxblnfk) [inline src/Driver/Postgres/Schema/PostgresColumn.php:280]: Please add a comment. Like "Internal field to determinate serial field is PK"

Comment (msmakouz) [reply src/Driver/Postgres/Schema/PostgresColumn.php:280]: Added

File: src/Driver/Postgres/Schema/PostgresColumn.php (hunk 1 above)
Comment (msmakouz) [reply src/Driver/Postgres/Schema/PostgresColumn.php]: Reverted

**Review by roxblnfk — APPROVED** [review]

**Pull request merged**